code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -314,3 +344,32 @@
 #undef MINOR_HEAL
 #undef MEDIUM_HEAL
 #undef MAJOR_HEAL
+
+
+
+/**
+ *Spittle; harmless reagent that is added by rat king, and makes you disgusted.
+ */
+
+/datum/reagent/rat_spit
+	name = "Rat Spit"
+	description = "Something coming from a rat. Dear god! Who knows where it's been!"
+	reagent_state = LIQUID
+	color = "#C8C8C8"
+	metabolization_rate = 0.03 * REAGENTS_METABOLISM
+	taste_description = "something funny"
+
+/datum/reagent/rat_spit/on_mob_metabolize(mob/living/L)
+	..()
+	to_chat(L, "<span class='notice'>This food has a funny taste!</span>")
+
+/datum/reagent/rat_spit/on_mob_life(mob/living/carbon/M)
+	if(prob(15))
+		to_chat(M, "<span class='notice'>That food was awful!</span>")
```

Comment:
ageusia check?

Reply:
It's supposed to cause rumblings in your stomach, like bad thai food. Ageusia would change nothing :C